src/a11y-dialog.ts

```diff
@@ -179,15 +179,10 @@ export default class A11yDialog {
    * (namely ESC and TAB)
    */
   private bindKeypress = (event: KeyboardEvent) => {
-    // This is an escape hatch in case there are nested dialogs,
-    // so the keypresses are only reacted to for the most recent one
-    if (!this.$el.contains(document.activeElement)) return
-
     // If the dialog is shown and the ESC key is pressed, prevent any further
     // effects from the ESC key and hide the dialog, unless its role is
     // `alertdialog`, which should be modal
     if (
-      this.shown &&
```

Comment:
🔍 Checking for `this.shown` is redundant here because this listener is bound and unbound when we show and hide, right?

Reply:
Right, I think that makes sense yes.